src/ActionRow/index.jsx

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
+
+const horizontalCSS = css`
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: flex-end;
+  & > * {
+    margin: 0;
+  }
+  & > * + * {
+    margin-left: .5rem;
+  }
+`;
+
+const stackedCSS = css`
```

Comment:
is nit-pick, but I feel like I would prefer the paradigm of an object of style values, rather than top-level variables...
ex: 
```
const styles = { 
  stacked: css``,
  horizontal: css``,
};

Reply:
updatedd